src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,9 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        if (number < 0) {
+            throw new IllegalArgumentException("number cannot less zero");
```

Comment:
change exception message to one from task description

Reply:
done